appveyor.yml

```diff
@@ -1,76 +0,0 @@
-version: 4.1.0.{build}
-clone_depth: 10
-install:
-build: off
-matrix:
-  fast_finish: true
-  exclude:
-    - platform: Ruby25-x64
-      LOCALE: ja-JP
-    - platform: Ruby24-x64
-      LOCALE: ja-JP
-
-# Ruby versions under test
-platform:
-  - Ruby26-x64
-  - Ruby25-x64
-  - Ruby24-x64
-  - Ruby23-x64
-
-# Note that locales are mainly code page changes and are not the FULL set of localization
-#  changes that happen when you install French/Germam/Japanese etc. Windows
-environment:
-  matrix:
-  - LOCALE: en-US
-  - LOCALE: ja-JP
-
-before_test:
-  # Semicolons are required below.  This renders as a single line powershell command in AppVeyor
-  - ps: >-
-      [string]$testLocale = [Environment]::GetEnvironmentVariable("LOCALE","Process");
-      $currentLocale = (Get-WinSystemLocale).Name;
-      if ($testLocale -ne '') {
-        if ($testLocale -ne $currentLocale) {
-          Write-Output "Setting Locale to $testLocale ...";
-          Set-WinSystemLocale ($testLocale);
-          Write-Output "Restarting worker in 5 seconds...";
-          Start-Sleep -Seconds 5
-          Restart-Computer -Force -Confirm:$false;
-        } else {
-          Write-Output "Locale is already set to $currentLocale";
-        };
-      } else {
-        Write-Output "LOCALE environment variable not set";
-      };
-
-test_script:
-  # The sleep is needed to allow the Appveyor tracing to start correctly after a reboot (locale change)
-  # - ps: Start-Sleep -Seconds 5
-  - timeout 6
-  - ps: |
-      chcp
-      Get-WinSystemLocale
-      $Env:PATH = "C:\${Env:PLATFORM}\bin;${Env:PATH}"
-      $Env:LOG_SPEC_ORDER = 'true'
-      Get-ChildItem Env: | % { Write-Output "$($_.Key): $($_.Value)"  }
-      # list current OpenSSL install
-      gem list openssl
-      ruby -ropenssl -e 'puts \"OpenSSL Version - #{OpenSSL::OPENSSL_VERSION}\"; puts \"OpenSSL Library Version - #{OpenSSL::OPENSSL_LIBRARY_VERSION}\"'
-      bundle install --jobs 4 --retry 2 --without packaging documentation
-      Get-Content Gemfile.lock
-      ruby -v
-      gem --version
-      bundle --version
```

Comment:
Lines 51-63 are useful when something goes wrong and there are environment differences. Could some of this be preserved (for both Windows and Linux)?